### PR TITLE
Apps-897 switch from cwltool to cwlpack

### PR DESCRIPTION
**Pull request by jdidion**

### CODE REVIEW

File: core/src/main/scala/dx/core/languages/cwl/CwlBlock.scala
Comment (YuxinShi0423) [inline core/src/main/scala/dx/core/languages/cwl/CwlBlock.scala:382]: @jdidion when unsatisfied is not empty, we call the orderSteps again, but those steps will still be recognized as unsatisfied this time, right?

Comment (r-i-v-a) [reply core/src/main/scala/dx/core/languages/cwl/CwlBlock.scala:382]: `deps = newDeps` will have added contents since the last call, so there is the chance that in each successive call, some more items will get partitioned into `satisfied` via 

```
case id if id.parent.exists(deps.contains) => true
```

Comment (YuxinShi0423) [reply core/src/main/scala/dx/core/languages/cwl/CwlBlock.scala:382]: Oh just realized it is not looping within one step. Thanks Riva!

Comment (jdidion) [reply core/src/main/scala/dx/core/languages/cwl/CwlBlock.scala:382]: no, because the dependencies have been updated

**Review by r-i-v-a — APPROVED** [review]

**Review by r-i-v-a — CHANGES_REQUESTED** [review]
Please add to release notes and fix what looks like repeated section at the top.

**Review by r-i-v-a — DISMISSED** [review]
Ignore earlier comment, I think there was a glitch in the GitHub review UI -- it is having problems handling all these files.

**Review by r-i-v-a — DISMISSED** [review]

**Review by r-i-v-a — APPROVED** [review]

**Comment by r-i-v-a** [comment]
I experienced some GitHub glitch while trying to merge this -- after clicking "Squash and merge," it stalled and didn't close the PR.

However, it looks like it's now successfully applied on develop -- there are 2 commits with this name, the first contains the actual contents from this PR and the second is empty.
-- https://github.com/dnanexus/dxCompiler/commit/bae9172ee010098c3f997ba476326938b1b59722
-- https://github.com/dnanexus/dxCompiler/commit/5915f947be1c5cce9c8105208b2657ecaf366ad9

**Pull request merged**
